catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -129,7 +171,14 @@ const insertIntoDir = (path: string[], file: FilesEntry, dir: FilesEntryDir) =>
   const newChildren = insertIntoTree(path, file, children)
   const type = newChildren
     .map(FilesEntry.match({ Dir: R.prop('type'), File: R.prop('type') }))
-    .reduce((acc, entryType) => (acc === entryType ? acc : 'modified'))
+    .reduce((acc, entryType) =>
+      // eslint-disable-next-line no-nested-ternary
```

Comment:
this rule is a part of the config we extend and i didnt want to add another override, but i think using ternaries deliberately makes sense. in this case i'll cave in to it and use ifs, tho i think it's a little more elegant with ternaries, because it is an expression and you dont have to use returns (tbh i think that's one of the most annoying part of js/ts which coffescript, where almost anything is an expression, got right)